Sources/PointFree/Pricing.swift

```diff
@@ -0,0 +1,190 @@
+import Css
```

Comment:
most of the stuff in this file can be ignored. it was mostly me experimenting with doing purely css tabs, and kinda snowballed from there. we'll come back to this soon.